VAMobile/documentation/docs/Engineering/DevOps/Creating a Firebase Project.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Creating a Firebase Project
+postion: 3
+---
+
+## Introduction
+
+This document will go through the steps of creating a firebase project.
```

Comment:
What's the situation where someone on our team would be using this documentation? If we're just documenting what we did to set up staging, I'm not sure we need this as that's more so docs Firebase would host